packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -14,61 +15,82 @@ import { SelectionMode } from 'office-ui-fabric-react/lib/Selection';
 import { Stack } from 'office-ui-fabric-react/lib/Stack';
 import { Text } from 'office-ui-fabric-react/lib/Text';
 import { ILinkToken } from 'office-ui-fabric-react/lib/common/DocPage.types';
-import { IApiInterfaceProperty, IApiEnumProperty, IMethod } from './ApiReferencesTableSet.types';
+import { IApiInterfaceProperty, IApiEnumProperty, IMethod, IApiReferencesTableProps } from './ApiReferencesTableSet.types';
 import { Markdown } from '../Markdown/index';
 import { getCurrentUrl } from '../../utilities/getCurrentUrl';
+import { PropertyType } from '../../utilities/parser/index';
+import { codeFontFamily } from '../CodeSnippet/CodeSnippet.styles';
 
-export interface IApiReferencesTableProps {
-  title?: string;
-  properties: IApiInterfaceProperty[] | IApiEnumProperty[];
-  methods?: IMethod[];
-  renderAsEnum?: boolean;
-  renderAsClass?: boolean;
-  renderAsTypeAlias?: boolean;
-  key?: string;
-  name?: string;
-  description?: string;
-  extendsTokens?: ILinkToken[];
-}
-
-export interface IApiReferencesTableState {
-  properties: IApiInterfaceProperty[] | IApiEnumProperty[];
-  methods?: IMethod[];
-  isEnum: boolean;
-  isClass: boolean;
-  isTypeAlias: boolean;
-}
+// TODO: remove
+export { IApiReferencesTableProps };
+export type IApiReferencesTableState = {};
 
 export const XSMALL_GAP_SIZE = 2.5;
 export const SMALL_GAP_SIZE = 8;
 export const MEDIUM_GAP_SIZE = 16;
 export const LARGE_GAP_SIZE = 48;
 
 const DEPRECATED_COLOR = '#FFF1CC';
+const rootClass = mergeStyles({
+  selectors: {
+    // Switch code blocks to a nicer font family and smaller size (monospace fonts tend to be large)
+    code: { fontFamily: codeFontFamily, fontSize: '11px' }
+  }
+});
+const theme = getTheme();
+const rowStyles: Partial<IDetailsRowStyles> = {
+  root: {
+    color: theme.semanticColors.bodyText,
+    selectors: {
+      ':hover': {
+        background: 'none',
+        color: theme.semanticColors.bodyText
+      }
+    }
+  },
+  isMultiline: {
+    wordBreak: 'break-word'
+  }
+};
 
+// When the text is passed to the function using this regex, it has had newline characters removed,
+// so the regex will match backtick sequences that span multiple lines.
```

Comment:
This comment is confusing; how are there multiple lines if there are no newline characters?

Reply:
I think it means sequences that originally spanned multiple lines? (I didn't write this, just moved it)